js_modules/dagit/packages/ui/src/components/Suggest.tsx

```diff
@@ -4,13 +4,16 @@ import {InputGroupProps2, IPopoverProps} from '@blueprintjs/core';
 import {isCreateNewItem, Suggest as BlueprintSuggest, SuggestProps} from '@blueprintjs/select';
 import deepmerge from 'deepmerge';
 import * as React from 'react';
-import {List} from 'react-virtualized';
+import {List as _List} from 'react-virtualized';
```

Comment:
I wonder if we should go ahead and switch this back to the tan-stack one soon. What were the problems you were having with the react-virtualized library on Windows?